### DISCUSSION TITLE
https://blog.marlam.in/passion/

### DISCUSSION THREAD

**Discussion by giscus** [Marlamin/blog#3] [Announcements]
# https://blog.marlam.in/passion/

There is 1 thing I will always make time for; passion. People talking about their passion, making something that comes from a place of passion or otherwise trying to follow whatever they're passionate about.

https://blog.marlam.in/passion/

**Comment by Marlamin** [comment]
Any particular things you happen to know I'm passionate about that you want to see? Have your own passion to share that you think I might be interested in? Let me know below!

_(I use GitHub Discussions to power comments on my blog, so you'll need a GitHub account to reply. If you don't have one, you can send me a tweet on Twitter as well.)_

**Reply by KalopsiaTwilight** [reply]
Hi Marlamin!

I think there's always been a special place in my heart for WoW private servers. After discovering what 'RP' realms were on retail, I somehow ended up on private RP servers when I was a teenager and eventually found a community full of lovely people that inspired me to become a better writer and person in general! The variety of custom servers that provide their own little takes on WoW as a game or story setting, I think is really cool and highlights how many people have played&loved WoW over the years and I love seeing new servers pop up here and there. I could go on a much longer rant but I'll try to keep things short here. I just think they're really neat as a creative and collaborative community effort. 

Right now I am involved with a private RP server and your work with WoW.tools has been invaluable for me to understand how a lot of the data tables fit together, so we would be able to customize things and I just wanted to take some time out of my day to thank you for that! I have enjoyed reading your blog posts and wish you all the best for the future! <3

**Reply by Marlamin** [reply]
That's awesome to read, thanks for taking the time to leave that comment. I'm glad the tools were/are helpful for you following your passion!

As for custom WoW content, WoW modding has a special place in my heart as well, regardless of it happening on a private server or just messing around with something like Noggit or even Blender locally. On a related note, I'm currently trying to push some of the modding tools/pipelines out of the dark ages (3.3.5) into the modern age. Progress is slow but steady and once things settle down a bit I might do a blog post on that specifically, even though the term "modding" is a bit controversial in some WoW circles. 😄

**Reply by KalopsiaTwilight** [reply]
Oooh! I'll be keeping my eye out for those in the future! I'd love to see more tools for more recent versions of WoW come out! :)
